test/instrumentation/drop-fast-exit-spans.test.js

```diff
@@ -66,6 +66,48 @@ tape.test('end to end test', function (t) {
   }, 20) // longer than exitSpanMinDuration
 })
 
+tape.test('end to end test with compression', function (t) {
+  resetAgent(function (data) {
+    // would normally be a single compressed span, but here there's
+    // no spans because min duration was not hit
+    t.equals(data.spans.length, 0)
```

Comment:
Could we change this test (or add a separate one) to check the case of a compressed span *not* being dropped, even if all of its included spans each have a duration that is less than `exitSpanMinDuration`?

Reply:
No objections if you wanted to take that on and think there's value there.